scylla-gdb.py

```diff
@@ -5395,10 +5395,11 @@ def dump_reads_from_semaphore(semaphore):
         permit_summaries = defaultdict(permit_stats)
         total = permit_stats()
 
+        schema_ptr_type = gdb.lookup_type('schema').pointer()
         for permit in intrusive_list(permit_list):
-            schema = permit['_schema']
+            schema = permit['_schema']['_p'].reinterpret_cast(schema_ptr_type)
```

Comment:
We strive to be backwards compatible with older scylla codebases. So this should be rewritten as:
```
try:
    schema = permit['_schema']['_p'].reinterpret_cast(schema_ptr_type)
except: gdb.error:
    schema = permit['_schema']
```

Reply:
Done. I was testing out why `schema_ptr` fails in my local setup using that second commit. I have dropped the second commit now.